### ISSUE TITLE
Wildcards Do Not Work in Negative Prompt

### ISSUE THREAD

**Issue by Quantumdrone** [adieyal/sd-dynamic-prompts#81]
**Describe the bug**
As of the last couple of weeks, wildcards do not work in the negative prompt. Normal prompting wildcards work just fine.

**Version used**
I see no Dynamic Prompts in "script dropdown", but the extensions tab tells me that it is the latest.

**To Reproduce**
1. Type any wildcard .txt file in your negative prompt, such as ```__colors__``` for colors.txt in the wildcard folder.
2. Generate image using prompt.
3. Check the image results and look at the embedded png info, and you will find that "```__colors__```" instead of something like "blue" or "yellow" is in the negative prompt.

As for my Dynamic Prompts settings, there is no change in options or checkboxes checked/unchecked from vanilla. Specifically, within the dropdown menu in the main generation tab, under Advanced Options, "Disable Negative Prompt" is unchecked.

In previous iterations of Auto's build of SD, wildcards in negative prompting works fine, but as of the past couple of weeks it no longer works.

**Comment by Quantumdrone** [comment]
My apologies. Misclick on my end.

**Comment by hardkur** [comment]
yes they worked for a little bit like a week before and now they are broken again

**Comment by adieyal** [comment]
There is a bug in auto1111 - I submitted a PR - https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/5251 which should fix it. If you can, please try this PR with the latest version of dynamic prompts and let me know if it still doesn't work for you.
